qiskit/circuit/quantumcircuit.py

```diff
@@ -4751,7 +4749,7 @@ def mcrx(
     def mcry(
         self,
         theta: ParameterValueType,
-        q_controls: Union[QuantumRegister, List[Qubit]],
+        q_controls: QuantumRegister | list[Qubit],
         q_target: Qubit,
         q_ancillae: Optional[Union[QuantumRegister, Tuple[QuantumRegister, int]]] = None,
         mode: Optional[str] = None,
```

Comment:
Sorry I didn't tag these before 🙂 I changed the typehint from `tuple[QuantumRegister, int]` to `list[Qubit]` because it looks like it should be a list of qubits, no?
```suggestion
        q_ancillae: QuantumRegister | list[Qubit] | None = None,
        mode: str | None = None,
```